README.md

```diff
@@ -12,28 +12,30 @@
 `etcd-druid` is an [etcd](https://github.com/etcd-io/etcd) [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) which makes it easy to configure, provision, reconcile and monitor etcd clusters. It enables management of an etcd cluster through [declarative Kubernetes API model](config/crd/bases/crd-druid.gardener.cloud_etcds.yaml). 
 
 In every etcd cluster managed by `etcd-druid`, each etcd member is a two container `Pod` which consists of:
+
 - [etcd-wrapper](https://github.com/gardener/etcd-wrapper) which manages the lifecycle (validation & initialization) of an embedded etcd.
 - [etcd-backup-restore](https://github.com/gardener/etcd-backup-restore) sidecar which currently provides the following capabilities (the list is not comprehensive):
   - [etcd](https://github.com/etcd-io/etcd) DB validation.
-  - Scheduled [etcd ](https://github.com/etcd-io/etcd)DB defragmentation.
+  - Scheduled [etcd](https://github.com/etcd-io/etcd) DB defragmentation.
   - Backup - [etcd](https://github.com/etcd-io/etcd) DB snapshots are taken regularly and backed in an object store if one is configured.
   - Restoration - In case of a DB corruption for a single-member cluster it helps in restoring from latest set of snapshots (full & delta).
   - Member control operations e.g. adding the peer `etcd` process as a learner.
 
 `etcd-druid` additional provides the following capabilities:
-* Facilitates declarative scale-out of [etcd ](https://github.com/etcd-io/etcd) clusters.
```

Comment:
Rest of the file uses `-` list syntax and here it is `*` syntax. Changing it in my PR due to this.